### DISCUSSION TITLE
Startup failed within offline environment

### DISCUSSION THREAD

**Discussion by Objia** [nextcloud/all-in-one#5741] [Questions] [answered]
Hi guys, I did not see the subject in github issues and on this reddit but I've been trying to install nextcloud all-in-one offline and by offline i mean without the host being able to join internet. I tried the following process :

Download all docker images from an "online" host ,Then bring them on the "offline" host with
"docker load -i nextcloud-images.tar"

Run a little docker-compose 

hen, docker-compose is complaining about "not being able to resolve nextcloud.com" but I do not care, our good friend nextcloud should have all the required images locally to be able to run without internet, am I not right ?

what should I do to solved this problem in offlline enviroment?

there is a link about this problem below,but without answers.
https://www.reddit.com/r/NextCloud/comments/13xouu8/is_it_possible_to_deploy_nextcloud_aio_offline/?rdt=47519

**Comment by Zoey2936** [comment, marked as answer]
https://github.com/nextcloud/all-in-one?tab=readme-ov-file#can-i-run-aio-offline-or-in-an-airgapped-system
